code/game/turfs/turf.dm

```diff
@@ -473,20 +474,32 @@
 	underlay_appearance.dir = adjacency_dir
 	return TRUE
 
-/turf/proc/add_blueprints(atom/movable/AM)
-	var/image/I = new
-	I.appearance = AM.appearance
-	I.appearance_flags = RESET_COLOR|RESET_ALPHA|RESET_TRANSFORM
-	I.loc = src
-	I.setDir(AM.dir)
-	I.alpha = 128
+///Certain sprites don't load properly upon init, so by queueing our orders we can wait until they have properly loaded in
```

Comment:
I asked action ninja to confirm, but wires probably autojoin during init, which means, the call that queues these, is going to happen at the wrong time.

https://github.com/tgstation/tgstation/blob/43ac7ae3b579dd1c5362d926a004920299e99085/code/game/objects/objs.dm#L68

Suspect the better way to do this, is instead of calling turf add_blueprints in object init, just put the item into a new list on the smoothing subsystem

https://github.com/tgstation/tgstation/blob/40d2a68f56ec59cccc316bb5a909cbbcbb7ba5ca/code/controllers/subsystem/icon_smooth.dm

Then, after smoothing has happened, that list can be iterated to build the blueprints items

tbh even now thinking about it you could even have it's own subsystem SSblueprint, who's sole job is just to init the list of blueprints after the rest of the subsystems.